CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   1. [#1337](https://github.com/influxdata/chronograf/pull/1337): Add support for blank hostnames on the Host List page
   1. [#1340](https://github.com/influxdata/chronograf/pull/1340): Fix case where the Explorer and cell editor falsely assumed there was no active query
   1. [#1338](https://github.com/influxdata/chronograf/pull/1338): Require url and name when adding a new source
+  1. [#1349](https://github.com/influxdata/chronograf/pull/1349): Add query for windows uptime
```

Comment:
This needs to be updated so the change is not in the section for the last release.